### DISCUSSION TITLE
custom key maps

### DISCUSSION THREAD

**Discussion by Flo4604** [scalar/scalar#7795] [Ideas]
### What happens?

This is in fact not a bug, see https://discord.com/channels/1135330207960678410/1268127488471732245/1268235804535296030

### What did you expect to happen?

It would be nice to be able to have custom keymaps maybe there could be some presets and then a settings page to change keybinds to your liking

### How can we reproduce the issue?

-

**Comment by amritk** [comment]
Hi @Flo4604 we actually do have the ability for custom keymaps now! We are just waiting for the settings page to be built so that you can actually set them
